plugins/Goals/TranslationHelper.php

```diff
@@ -14,6 +14,33 @@
 
 class TranslationHelper
 {
+
+    public function getTranslationForGoalMetricCategoryText($type, $category)
```

Comment:
It's hard to understand what the method name means, maybe it could be simplified a little? Something like `translateGoalMetricCategory`?